azure-communication-ui/call-with-chat/src/main/java/com/azure/android/communication/ui/callwithchat/CallWithChatCompositeBuilder.java

```diff
@@ -7,8 +7,8 @@
 import com.azure.android.communication.ui.calling.CallCompositeBuilder;
 import com.azure.android.communication.ui.callwithchat.models.CallWithChatCompositeLocalizationOptions;
 import com.azure.android.communication.ui.callwithchat.service.CallWithChatService;
-import com.azure.android.communication.ui.chat.ChatUIClient;
-import com.azure.android.communication.ui.chat.ChatUIClientBuilder;
+import com.azure.android.communication.ui.chat.ChatAdapter;
+import com.azure.android.communication.ui.chat.ChatAdapterBuilder;
 
 public final class CallWithChatCompositeBuilder {
```

Comment:
Should it be public?

Reply:
Don't worry about this code, it's in hibernation (CallWithChat library code). It's not part of the release.